src/kernels/webgl/argminmax_gpu.ts

```diff
@@ -44,7 +44,7 @@ export class ArgMinMaxProgram implements GPGPUProgram {
         int outIdx = coords[1];
         int inOffset = outIdx * ${windowSize};
 
-        int bestIndex = 0;
+        int bestIndex = inOffset;
```

Comment:
This would need a test which you can add [here](https://github.com/tensorflow/tfjs-core/blob/master/src/ops/reduction_ops_test.ts#L415)

Reply:
Hey, thanks for the suggestion. I went ahead an added a test case for both argMin and argMax which this change affects.